index.html

```diff
@@ -1388,16 +1388,17 @@ <h2>
       </pre>
 
       <p class="note" title="Determining the DID controller">
-If the <code><a>controller</a></code> property is not present, the
-<a>DID controller</a> can be determined by consulting the respective
-<a>DID method</a> specification. It cannot be assumed that the
-<a>DID subject</a> is also the controller of its own <a>DID</a>, nor
-can it be assumed that <em>only</em> entities identified by <a>DIDs</a>
-present in the value of <code><a>controller</a></code> are capable of exerting
-control over the <a>DID</a> in question. Similarly, <a>DID methods</a> are not
-obliged to respect the value of <code><a>controller</a></code> and can specify
-means for updating the <a>DID document</a> that are unrelated to the
-<code><a>controller</a></code> property.
+If the <code><a>controller</a></code> property is present, its value expresses
+the root of authority for information expressed in the <a>DID Document.</a> If
+the <code><a>controller</a></code> property is not present, the value can be
+inferred to be the same as that of the <code><a>id</a></code> property of the
+<a>DID document</a>. The way to determine how the <a>DID Document</a> can be
+changed is by consulting the relevant <a>DID method</a>. If no <a>verification
+methods</a> are present in the <a>DID Document</a>, then a
+<code><a>controller</a></code> property MUST be present, otherwise the
+the <a>DID document</a> can no longer be changed beyond whatever its present
+value is.
```

Comment:
@jandrieu -- would this address your concern?

```suggestion
<code><a>controller</a></code> property MUST be present, otherwise it
is possible that the <a>DID document</a> can no longer be changed beyond
whatever its present value is. This can be determined by consulting the
applicable <a>DID method</a> specification.
```

Reply:
It seems that the "MUST" is misplaced. If "MUST" is used, that's a normative requirement, which means the following phrase of "otherwise it is possible" is not actually possible with a conformant implementation.

Are you intending to say that, as a requirement,
* DID Methods MUST return DID Documents that either have a verification method or a controller property. 
* DID Methods MUST respect the controller property

That's what this language suggests to me. 

However, I don't believe BTCR does either of this today (I could be wrong), which makes me wonder if this particular use is actually in sync with a minimal set of capability for DIDs as envisioned to date.

Reply:
I thought this while I was drafting it and then apparently completely forgot: we can't use normative language in a note block, so this MUST needs to come out anyway.

Reply:
I'm fine if this is reworded to remove the "MUST" since it's in a note. The language I'm looking for here is an indication to DID-method agnostic consumers (such as verification software) such that it can understand the root of authority over what is expressed in the DID Document. This should either be the `controller` (if present) or the `subject`, IMO. Whether or not DID methods have implementation specific details for updating DID Documents is a separate matter to me. I want consuming software to be able to be structured such that it only need to decide whether or not it trusts a given DID method/resolver for that method and then from there, once it has a DID Document, it can assume the root authority is as stated above (thereby allowing consistent verification code).